fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -642,7 +644,10 @@ private boolean processModifyColumn(ModifyColumnClause alterClause, OlapTable ol
             }
         } // end for handling other indices
 
-        return false;
+        if (modColumn.isKey() || !modColumn.getType().isScalarType()) {
```

Comment:
why modify column does not support fast schema evolution?

Reply:
Modifying a key column involves the encoding of a short key, which is relatively complex and is not supported for the time being.